components/Tabs/index.js

```diff
@@ -7,3 +7,66 @@
 //
 //  The tab component should look like this:
 //    <div class="tab">topic here</div>
+
+//const tabs= document.querySelector('.tabs');
+
+const topicsDiv = document.querySelector('.topics');
+
+const topicsArray = ['JAVASCRIPT', 'BOOTSTRAP', 'TECHNOLOGY', 'JQUERY', 'NODE.JS'];
+
+topicsArray.forEach( topic => {
+    axios.get('https://lambda-times-backend.herokuapp.com/topics')
+    .then(response => {
+        console.log(response);
```

Comment:
if this returns an array of topics. when looping through them is there a function that would be looking to accept a argument for a title?

Reply:
what is creating the tabs